docs/manual/view_management.rst

```diff
@@ -280,22 +285,33 @@ Zoom/Z-axis Likewise works the same as Tray_ mode.
   pointing at the same location.
 
 
-.. figure:: view_controls/Space1.png
-   :width: 45%
+.. |space1| image:: view_controls/Space1.png
 
-   Rotation in the vertical path does not have stoppers.
+   
+.. |spacetext1| replace:: Rotation in the vertical path does not have
+   stopping points.
+   
+.. |space2| image:: view_controls/Space2.png
 
-.. Figure:: view_controls/Space2.png
-   :width: 45%
+.. |spacetext2| replace:: Left & Right rotation are relative to the
+   current view direction.
 
-   Left & Right pan are relative to the current view direction.
+.. |space3| image:: view_controls/Space3.png
 
-.. figure:: view_controls/Space3.png
-   :width: 45%
+.. |spacetext3| replace:: Space mode can barrel roll! (see the `mouse
+   control table. <mousecontrols_>`_) Up & Down rotation are relative
+   to the current view direction.
```

Comment:
Why all these substitution definitions? It's a serious pain to format text to fit into the cells in the row-based format, and would make diffs for any future changes to phrasing hard to read.

This also makes it simpler to re-order the table any time we choose.

It's not nearly so bad in the column-based version above.

Is the non-locality worth it?

Reply:
Gahh! The inline diff look horrid. Please click on the revision-number to get the side-by-side version. It makes much more sense!

Reply:
> Why all these substitution definitions?

The substitutions style is the only thing that makes the least bit sense here. The table definition in .rst is just stupid to begin with.

Reply:
Well, any other table definition setup that I've ever seen required lots of markup tags... Think of a `.html` `<tr><td></td></tr>` nest.... Not very readable, whereas the `.rst` format is at least human-understandable.